### ISSUE TITLE
odd rendering of the "full block" starting in recent updates

### ISSUE THREAD

**Issue by aaalexand** [braver/fonts#98]
This could be one of many things, but a recent update to this package has made it difficult to use certain fonts, particularly (but not exclusively) those with ligature support. I like to use the minimap-titles package to quickly make ASCII art using FIGlet. The problem is that the "U+2588 : FULL BLOCK {solid}" character no longer seem to be rendering appropriately.

Here's what Fira Code does now.
![image](https://user-images.githubusercontent.com/25070358/74357357-ce97be00-4d8d-11ea-8cc9-6d796244415e.png)

Here's Source Code Pro (looks good!)
![image](https://user-images.githubusercontent.com/25070358/74357539-11f22c80-4d8e-11ea-927d-45769dfc349d.png)

And here's a bonus issue occurring in Inconsolata (I'm not sure if this was happening before though).
![image](https://user-images.githubusercontent.com/25070358/74357814-79a87780-4d8e-11ea-8c9d-151b68e582fe.png)

I've tried basic end-user things like fiddling with different themes for UI/syntax, and poking around the config file, but I'm not sure what altered the rendering of some of these characters in the first place.

**Comment by lierdakil** [comment]
Hmm. Unless by "recent" you mean v3.4.0, which was released almost a year ago, I don't have any ideas what might have changed.

FWIW, FiraCode works fine on my end (Atom v1.43.0, fonts v3.10.0):
![image](https://user-images.githubusercontent.com/7275622/74361370-d67c5000-4dd7-11ea-930c-77a80579a575.png)

I have this in my stylesheet:
```css
atom-text-editor {
  font-feature-settings: "zero", "ss01", "ss03", "ss06";
  font-variant-ligatures: normal;
}
```

Also of note, Inconsolata seems to "just be like that", at least in the [programming fonts test drive app](https://app.programmingfonts.org/#inconsolata) it behaves in a similar fashion:
![image](https://user-images.githubusercontent.com/7275622/74361521-21966300-4dd8-11ea-8cd6-42d670eeaf1d.png)

Inconsolata-g is fine though.

For here's the text I used for testing:
```
█     █    █     ████   ███   ████ 
██   ██   █ █   █    █   █   █    █
█ █ █ █  █   █  █        █   █     
█  █  █  █████  █  ███   █   █     
█     █  █   █  █    █   █   █    █
█     █  █   █   ████   ███   ████ 
```
